### ISSUE TITLE
Fix Spruce streaming in Browserstack

### ISSUE THREAD

**Issue by cjcolvar** [avalonmediasystem/avalon#3729]
Testing with android and iOS devices in browserstack fail to stream from spruce.  This makes it hard to do QA and regression testing.

**Comment by joncameron** [comment]
This may become priority if we become stymied during testing and need to rely on BrowserStack. We can also push code to Mallorn for testing if need be and Spruce is still having issues.

**Comment by cjcolvar** [comment]
This came up again when testing for #3932.  @phuongdh explained to me that the issue is that port 8980 is firewalled for mallorn and spruce.  Some possible solutions to this issue are:
1. Do browserstack testing on an existing public instance like mco-staging or demobox.
2. Ask UITS to unblock that port for mallorn and spruce.
3. Spin up a turnkey server for browserstack testing when needed and then tear it down when done.

**Comment by phuongdh** [comment]
It'd be great to modify Turnkey so it can deploy multiple Avalon instances on the same stack, then we can test deploy any branch whenever it's needed.

**Comment by SumithBaddam** [comment]
I was testing #3562 on browser stack. Though I am connected to VPN I am not able to play media objects on Spruce from any device. It was displaying just a blank screen (unlike displaying media error or something when not connected to VPN). When I tried to look at the Network logs from dev tools, I observed that, when run on laptop, the index media file auto.m3u3?token....is getting status 200 and next time 304 no modification. But from browserstack it is just 304. There is no 200 success token. So I doubt if it is getting fetched from the API correctly.
